modules/environment-roles/root.tf

```diff
@@ -185,8 +185,26 @@ resource "aws_iam_role" "custodian_deploy_role" {
   )
 }
 
+resource "aws_iam_role" "custodian_github_actions_deploy_role" {
```

Comment:
Maybe rename this: `github_actions_custodian_deploy_role` to make it consistent with the other github actions role resources names?

Reply:
Yep, I've changed it here and the role attach resource as well.